contracts/src/v0.8/ccip/pools/GHO/UpgradeableBurnMintTokenPool.sol

```diff
@@ -58,9 +59,4 @@ contract UpgradeableBurnMintTokenPool is VersionedInitializable, UpgradeableBurn
   function REVISION() public pure virtual returns (uint256) {
```

Comment:
while this doesn't hurt it also doesn't help and is a bit useless when using `Initializable` imo. I'd rather remove it.

Reply:
removed, ty!